js/customize-controls.js

```diff
@@ -12,7 +12,7 @@
 
 		// Only show the color hue control when there's a custom color scheme.
 		wp.customize( 'colorscheme', function( setting ) {
-			wp.customize.control( 'colorscheme_hue', function( control ) {
+			wp.customize.control( 'colorscheme_primary_hue', function( control ) {
```

Comment:
@kjellr Fixed a small typo here so this should be good to go now.

Reply:
👍